src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -145,6 +150,20 @@ enum class SvgParserLengthType
     Other
 };
 
+enum BBoxFlag
+{
+    None = 0,
+    Width = 1,
+    Height = 2,
+    ViewboxX = 4,
+    ViewboxY = 8,
+    ViewboxW = 16,
+    ViewboxH = 32,
```

Comment:
How about displaying as 0x1, 0x2 referring to `enum class SvgStyleFlags`?

Reply:
done